service.kn.switchtimer/service.py

```diff
@@ -134,44 +134,50 @@ def poll(self):
 
                 if _timer['utime'] < _now: _switchInstantly = True
                 if (_timer['utime'] - _now < INTERVAL + self.__dispMsgTime / 1000) or _switchInstantly:
-                    chanIdTmr = self.channelName2channelId(_timer['channel'].decode('utf-8'))
+                    chanIdTmr = self.channelName2channelId(_timer['channel'])
                     if chanIdTmr:
 
                         # get player properties, switch if necessary
 
                         plrProps = self.getPlayer()
                         if chanIdTmr == plrProps['id']:
                             handler.notifyLog('Channel switching unnecessary')
-                            handler.notifyOSD(__LS__(30000), __LS__(30027) % (_timer['channel'].decode('utf-8')), time=self.__dispMsgTime)
+                            handler.notifyOSD(__LS__(30000), __LS__(30027) % (_timer['title'], _timer['channel']), time=self.__dispMsgTime)
```

Comment:
Just a quick note. We are discouraging the use of \_\_label\_\_ style labels. They should only be used for Python internals, not variable names.

Reply:
I had a small talk about this with phil65 for a longer time ago. As I was learning Python in 2011 from the coder examples here, this was the 'default' code style. So I've learned this from here - thinking this is okay then ;)

Reply:
Yup, I learned this from @phil65 as well. I slowly changed most of my add-ons (40 or 50 of them) to eliminate as I did upgrades, etc. Your code has been merged, the above was just a reminder. For the exact reason that you give above, people use the kodi.org repo as examples, it's good if we can encourage better style.